resources/osm_importer/webots_objects/road.py

```diff
@@ -113,21 +113,27 @@ def parse_tags(self):
         if 'layer' in self.tags:
             self.layer = float(self.tags['layer'])
 
-        if 'lanes:forward' in self.tags:
-            self.forwardLanes = int(self.tags['lanes:forward'])
-            self.backwardLanes = self.lanes - self.forwardLanes
-            assert self.forwardLanes > 0
-            assert self.lanes - self.forwardLanes >= 0
-
-        if 'lanes:backward' in self.tags:
-            self.backwardLanes = int(self.tags['lanes:backward'])
-            # note: "lanes" is overriden if "lanes:backward" and "lanes:forward" are both defined.
-            if ('lanes:forward' in self.tags):
-                self.lanes = self.forwardLanes + self.backwardLanes
-            else:
-                self.forwardLanes = self.lanes - self.backwardLanes
-            assert self.backwardLanes > 0
-            assert self.lanes - self.backwardLanes >= 0
+        try:
+            if 'lanes:forward' in self.tags:
+                self.forwardLanes = int(self.tags['lanes:forward'])
+                self.backwardLanes = self.lanes - self.forwardLanes
+                assert self.forwardLanes > 0
+                assert self.lanes - self.forwardLanes >= 0
```

Comment:
Are these assert still useful now that they are in a try-except ?

Reply:
I think yes, because the try-except checks if it's an int, but the assert checks it's positive. However asserts are not ideal because they stop the execution of the program.

Reply:
Yes indeed but will the assert will have an effect inside try-except ?

Reply:
A Python `assert` throws an `AssertionError`, and not a `ValueError` which is catched here. So the behavior remain unchanged.

However, I think we should deal these 2 cases the same way, probably by not stopping the execution of the program, and display a simple warning rather.

Reply:
Thank you for the explanation, you are completely right, I did indeed miss the `ValueError`.